test/src/bgw/params.c

```diff
@@ -99,19 +99,26 @@ params_get_dsm_handle()
 }
 
 static TestParamsWrapper *
-params_open_wrapper()
+params_open_wrapper(bool *do_close)
```

Comment:
nit: maybe `openned`?

Reply:
just trying to be consistent with the corresponding `params_close_wrapper` call ;-)